CMakeLists.txt

```diff
@@ -551,8 +551,8 @@ if (WIN32)
     "${DDK_ROOT}/tools/tracing/${DDK_SFX}/dbghelp.dll"
     "${PROGFILES}/Microsoft Visual Studio */Common7/IDE/Remote Debugger/${ARCH_SFX}/dbghelp.dll"
     "${PROGFILES32}/Windows Kits/*/Debuggers/${ARCH_SFX}/dbghelp.dll"
-    # For Visual Studio 2010+, x64 dbghelp.dll resides in Program
-    # Files (x86) rather than Program Files.
+    # In case if SDK is not installed and we have Visual Studio, dbghelp.dll may be found in the
+    # Visual Studio's directory in subfolder Remote Debugger/x86 or /x64 folders (xref i#1956).
```

Comment:
Yes.  This is what I see on a machine with many versions installed:

```
/c/Program Files/Microsoft Visual Studio 8/Common7/IDE/Remote Debugger/x64/dbghelp.dll
/c/Program Files/Microsoft Visual Studio 9.0/Common7/IDE/Remote Debugger/x64/dbghelp.dll
/c/Program Files/Microsoft Visual Studio 9.0/Common7/IDE/Remote Debugger/x86/dbghelp.dll
/c/Program Files (x86)/Microsoft Visual Studio 10.0/Common7/IDE/Remote Debugger/x64/dbghelp.dll
/c/Program Files (x86)/Microsoft Visual Studio 10.0/Common7/IDE/Remote Debugger/x86/dbghelp.dll
/c/Program Files (x86)/Microsoft Visual Studio 11.0/Common7/IDE/Remote Debugger/x64/dbghelp.dll
/c/Program Files (x86)/Microsoft Visual Studio 11.0/Common7/IDE/Remote Debugger/x86/dbghelp.dll
/c/Program Files (x86)/Microsoft Visual Studio 12.0/Common7/IDE/Remote Debugger/x64/dbghelp.dll
/c/Program Files (x86)/Microsoft Visual Studio 12.0/Common7/IDE/Remote Debugger/x86/dbghelp.dll
/c/Program Files (x86)/Microsoft Visual Studio 8/Common7/IDE/Remote Debugger/x86/dbghelp.dll
```

I guess 2008 is putting 32-bit in progfiles64 too.

The upshot is, the simplest thing is to look in *both* progfiles32 (the (x86) one) and progfiles64 (no parens) b/c of this variation across versions.